diff -ruN orig/App.pro new/App.pro
--- orig/App.pro	2017-10-25 05:30:49.000000000 -0600
+++ new/App.pro	2018-07-18 11:55:09.319372058 -0600
@@ -52,7 +52,7 @@
     svgattributeeditor.h
 
 linux-g++ {
-    LIBS += -L$$PWD/lib/
+    LIBS += -L$$PWD/lib64/
     LIBS += -lm
     LIBS += -ldl
 }
diff -ruN orig/AppRun new/AppRun
--- orig/AppRun	2017-10-25 05:30:49.000000000 -0600
+++ new/AppRun	2018-07-18 11:52:55.440995393 -0600
@@ -1,4 +1,4 @@
 #!/bin/bash
 HERE="$(dirname "$(readlink -f "${0}")")"
-export LD_LIBRARY_PATH=${HERE}/usr/lib:$LD_LIBRARY_PATH
+export LD_LIBRARY_PATH=${HERE}/usr/lib64:$LD_LIBRARY_PATH
 exec "${HERE}/usr/bin/AnimationMaker" "$@"
diff -ruN orig/Widgets/Widgets.pro new/Widgets/Widgets.pro
--- orig/Widgets/Widgets.pro	2017-10-25 05:30:49.000000000 -0600
+++ new/Widgets/Widgets.pro	2018-07-18 11:54:06.906730271 -0600
@@ -78,6 +78,6 @@
     installer.h
     
 unix {
-    target.path = /usr/lib
+    target.path = /usr/lib64
     INSTALLS += target
 }
